parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -467,6 +467,79 @@ renderTypeError e env src curPath = case e of
         annotatedAsErrorSite src abilityCheckFailureSite,
         debugSummary note
       ]
+  AbilityEqFailure {..} ->
+    mconcat
+      [ "I could not solve an ability equation when checking the expression ",
+        describeStyle ErrorSite,
+        "\n\n",
+        Pr.wrap $
+          mconcat
+            [ "When trying to match ",
```

Comment:
```suggestion
            [ "when trying to match ",
```

Reply:
am neutral on this wording change, going to leave it as is

Reply:
Elaborating a bit, the start of the sentence already establishes the compiler as talking in first person, which is the style we like for our messages. But no need to insert first person in every clause of the sentence and in this particular case I'm not sure it reads any better.

"I found a mismatch here... while I was trying to..." 
vs 
"I found a mismatch here... when trying to..."